cryptoconditions/types/threshold_sha256.py

```diff
@@ -562,8 +627,8 @@ def validate(self, message=None, **kwargs):
         #     # Fulfillment is not minimal
         #     return False
         # TODO: ILP specs see unfulfilled conditions as conditions and not fulfillments
-        valid_decisions = []
+        valid_decisions = 0
         for fulfillment in fulfillments:
             if fulfillment['body'].validate(message, **kwargs):
-                valid_decisions += [True] * fulfillment['weight']
```

Comment:
;-)

That's indeed a valid shortcut
PS: ILP/CC is probably removing weights in the near future